dash/include/dash/Meta.h

```diff
@@ -90,6 +90,19 @@ struct is_container_compatible :
          >
 { };
 
+/**
+ * Type trait indicating whether a type can be used for global atomic
+ * operations.
+ */
+template <typename T>
+struct is_atomic_compatible
+: public std::integral_constant<
+           bool,
+              dash::is_container_compatible<T>::value
+           && sizeof(T) <= sizeof(std::size_t)
+         >
```

Comment:
@fmoessbauer Absolutely! But currently, a DART reduce operation would not work for types with size greater than 64 bit, right?
Would compare-exchange on `dash::Atomic<T>` currently work for `T = struct { short foo[128] }` ?